.github/workflows/update_prebuilts.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow creates a new PR with the updated build ids for Playground prebuilts.
+# To run it, navigate to https://github.com/androidx/androidx/actions/workflows/update_prebuilts.yml
+# and click "Run workflow"
+name: Update Prebuilts
+on:
+  workflow_dispatch:
+    inputs:
+      target-branch:
+        description: 'The target branch for the PR'
+        required: false
+        default: 'androidx-main'
+        type: string
+      comment:
+        description: 'Any comment you would like to add to the PR body'
+        required: false
+        default: ''
+        type: string
+jobs:
+  update-prebuilts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout androidx repo"
+        uses: actions/checkout@v2
+      - name: "Set environment variables"
+        shell: bash
+        run: development/update_playground.sh
+      - name: "Check if anything changed"
+        shell: bash
+        id: "check-for-changes"
+        run: |
+          set -x
+          git status
```

Comment:
is this just for logging? with set -x i don't think we'll need this to see the output of git status since we also call it below

Reply:
yea just to see it but also below we pipe it, idk if set x will show every intermediate step's output.